test/data/manifests/fedora-ostree-oci-bootiso.mpp.yaml

```diff
@@ -0,0 +1,428 @@
+version: '2'
+pipelines:
+  - mpp-import-pipelines:
+      path: fedora-vars.ipp.yaml
+  - mpp-import-pipeline:
+      path: fedora-build-v2.ipp.yaml
+      id: build
+    runner:
+      mpp-format-string: org.osbuild.fedora{release}
+  - name: anaconda-tree
```

Comment:
It seems like 80% of this is just "build an anaconda image" that is OS independent and should be factored out?